metaswitch/common/stats_to_dita.py

```diff
@@ -204,3 +204,22 @@ def should_output_stat(stat_name):
                         file_oid_name,
                         table_oids,
                         stats)
+
+
+def oid_sort_key(oid):
+    """Sorts OIDs lexicographically.
+
+    `oid` is a string containing an OID in numeric form e.g.
+    ".1.2.3" or "342.3.2".
+    For use as a key function in sorting tools like `sorted`."""
+    # OIDs:
+    # * May start with an optional '.'
+    # * Are a sequence of decimal characters separated by '.'s.
```

Comment:
Should this just be part of the function's doc string? You are basically describing the valid input here.